packages/postcss-merge-rules/src/index.js

```diff
@@ -307,7 +307,7 @@ function partialMerge(first, second) {
 
   /**
    * @param {function(import('postcss').Declaration):void} callback
-   * @this {import('postcss').Rule}
+   * @this void
```

Comment:
Now TypeScript sees it as void. I think it could be right since it's arrow functions all the way, so maybe `this` never gets bound, but I don't think it matters since `this` it's never used in the callbacks either.